configs/nextflow.base.config

```diff
@@ -147,17 +147,20 @@ process {
 	withName: add_igv_track {
 		container = "${params.containerDir}/bonsai-prp.sif"
 		publishDir = [ path: "${params.outdir}/${params.speciesDir}/analysis_result", mode: 'copy', overwrite: true ]
+		ext.when = { params.species == "mycobacterium tuberculosis" }
 	}
 	withName: amrfinderplus {
 		container = "${params.containerDir}/ncbi-amrfinderplus.sif"
 		cpus = 4
 		memory = '10.GB'
 		publishDir = [ path: "${params.outdir}/${params.speciesDir}/amrfinderplus", mode: 'copy', overwrite: true ]
 		ext.args = "--plus"
+		ext.when = { params.species != "mycobacterium tuberculosis" }
```

Comment:
Should the execution of a module be controlled by the species param? 

Would an option be to instead use a `use_amrfinder` boolean to trigger an amrfinder analysis?

If we have two assays where we dont run amrfinder+, would the argument be something like this?
```
ext.when = { params.species != "mycobacterium tuberculosis" || params.species != "vibrio cholerae" }
```

Reply:
This is a good point, problem is that there will have to be a `use_process` argument for every process. Which we will have to include in every profile. I'm not sure which is the lesser of two evils.